src/pycpt/notebook.py

```diff
@@ -1008,7 +1010,7 @@ def plot_mme_flex_forecasts(
 
     # plot exceedance probability map
 
-    ForTitle, vmin, vmax, mark, barcolor = ce.prepare_canvas("POE", predictand_name)
+    ForTitle, vmin, vmax, mark, barcolor = ce.prepare_canvas('POE',predictand_name,None,color_bar)
```

Comment:
Use a keyword argument so that you don't have to specify `type`:
```
ce.prepare_canvas('POE', predictand_name, color_bar=color_bar)
```

Note that adding `color_bar` is a backwards-incompatible change (won't work with versions of cptextras prior to https://github.com/iri-pycpt/CPT-EXTRAS/pull/6/files), and we need to indicate that in the package metadata.   We can't do that until the new cptextras package is released, so we'll wait to release this until that is done.

Reply:
@xchourio could you address this comment so I can release your change?

Reply:
Done!